source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,66 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes to data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. 
+
+The ``watch()`` method takes an optional pipeline as the first parameter
+that is an array of stages to filter the :manual:`change events
+</reference/change-events/>` output. Also, ``watch()`` can take an
+additional options object as the second parameter. Set the ``fullDocument``
+field of the additional options object to ``'updateLookup'`` to receive
+an event that contains the changes to the document as well as the entire
+altered document. Alternatively, you can set ``fullDocument`` to the
+string ``'default'`` to only receive the altered document, but not the
+changes to the document. 
+
+Process the Change Stream Events
+--------------------------------
+
+You can capture events from a change stream instance by using a listener
+function. Call the watch() command to get a Change Stream instance. Add
+your listener function by calling the `EventEmitter.on()
+<https://nodejs.org/api/events.html#events_emitter_on_eventname_listener>`_
+method on that instance. Set the value of the first parameter to the
+string ``'change'`` and add your :mdn:`callback function
+<Glossary/Callback_function>` as the second parameter.  The callback
+triggers when a change event is emitted, providing the next available
+document. You can specify logic in the callback to process the event
+document when it is received. 
+
+Call the :node-api:`close() </ChangeStream.html#close>` method on the
+change stream instance to stop processing change events. This method
+closes the change stream and frees resources.
+
+Example
+-------
+
+The following example opens a ``Change Stream`` on the ``movies``
+collection. We create a ``changeListener`` variable that is listener
+function to receive and print change events that occur. 
+
+To emit a change event, we want to perform a change to the collection, such as insertion
+with ``insertOne()``. To prevent the ``insertOne()`` from executing
+before ``changeListener`` can register, we create a ``firstTimer``
+variable that uses :mdn:`setTimeout
+<Web/API/WindowOrWorkerGlobalScope/setTimeout>` to wait 1 second. After
+the insertion of the document, the listener should receive this event.
+
+After that, we want to close the ``Change Stream`` instance.  We can do
```

Comment:
Nitpick: You can omit "want to"  and "can do this by" since the rest of this description is not about building up code, but rather describing what is there. How about:
"We close the ``Change Stream`` instance using ``changeStream.close()`` one second after we insert the document to allow the event to trigger the listener.

Reply:
I disagree that the description is about building up code. The description is solely a tool to help readers understand the code through whatever means (whether that is describing the code or talking about how the code is built). 

Some things I like about my current version:
- "After that, we want to close the Change Stream instance. We can do this by creating a second timer to wait for an additional second ..." is more explanatory than "We close the Change Stream instance using ``changeStream.close()`` one second after"
- It's less run-on than the suggestion

I don't think either is right or wrong, in fact, I think both communicate the same thing to a reasonable degree of effectiveness. As the writer of the doc, I'd like to keep it as is.

Reply:
In my comment, I mentioned "the description is not about building up code" which I think you agree with. The request was to omit "we can do this by".

Reply:
Sorry I meant: "I disagree that the description is **not necessarily** about building up code." 

But I'm game to change it to the sugesstion, see https://github.com/mongodb/docs-node/pull/52#discussion_r377282082 for further thoughts